ros_support_interfaces/uc2_interfaces/action/InspectBlock.action

```diff
@@ -0,0 +1,7 @@
+# action pattern
+# attributes of goal
+int32 block_id

```

Comment:
All of
- InspectBlock.action
- InspectBlockImplem.action
- MoveBlock.action
- MoveBlockImplem.action
- RecoverBlock.action **AND**
- RecoverBlockImplem.action

are exactly the same. Why not use one definition for all?

Reply:
This is a question for @m-morelli , but I would expect the original interfaces (with strings and so on) to have some differences...